### ISSUE TITLE
Short Form in YAML not Interpreted Properly

### ISSUE THREAD

**Issue by johnraws** [awslabs/aws-cfn-lint-visual-studio-code#26]
When I open a YAML template file that contains short form functions, the following Problems are reported:

unknown tag <!Ref>
unknown tag <!FindInMap>
unknown tag <!Sub>

Example snippet below, causing the first Problem shown above.
```
Parameters:
    pNotifyEmail:
        Description: Notification email
        Type: String
        Default: ''
Resources:
    rSecurityAlarmTopic:
        Type: AWS::SNS::Topic
        Properties:
            Subscription:
              - Endpoint: !Ref pNotifyEmail
                Protocol: email
```

I have ensured the newest version is installed via PIP:
```
>cfn-lint -v
cfn-lint 0.7.4
```

Please let me know if there is any additional information that can be provided to assist with troubleshooting.

**Comment by kddejong** [comment]
I believe these errors are given by a tool other than cfn-lint.  This template didn't provide me the listed errors until I installed the package `yaml support by redhat`  Those additional packages will need to be configured to support the shorthand functions made available in CloudFormation yaml.

**Comment by johnraws** [comment]
Great catch! I removed that extension and the error has been removed from my Problems view. Closing the Issue.
